test/e2e/lib/helpers/browser.ts

```diff
@@ -1,8 +1,10 @@
 import * as os from 'os'
 
-// typing issue for execute https://github.com/webdriverio/webdriverio/issues/3796
-export function browserExecute(fn: any) {
-  return browser.execute(fn)
+export function browserExecute<ReturnValue, InnerArguments extends any[]>(
+  fn: (...args: InnerArguments) => ReturnValue,
+  ...args: InnerArguments
+): Promise<ReturnValue> {
+  return browser.execute(fn, ...args)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I needed the `InnerArguments`. However, because the linked issue seems to be closed, I propose to refactor (in a separate PR) the tests to use `browser.execute()` directly. wdyt?

Reply:
Sounds good to me

Reply:
here: https://github.com/DataDog/browser-sdk/pull/2700